### ISSUE TITLE
Tips on opening this file?

### ISSUE THREAD

**Issue by chconnor** [domlysz/BlenderGIS#127]
Hi -- I'm trying to open a file and am getting an error. The file is [here](http://caseyconnor.org/pub/image/example.tif). (~9MB).

I try to import it as "Raw DEM", build faces is checked, step can be whatever, clip to working extent is unchecked, and CRS can be whatever. When I try the import, I get this traceback in the Info window:

```
location: <unknown location>:-1

Traceback (most recent call last):
  File "/home/casey/.config/blender/2.79/scripts/addons/BlenderGIS-master/operators/io_import_georaster.py", line 403, in execute
    mesh = exportAsMesh(grid, dx, dy, self.step, reproj=rprjToScene, subset=self.clip, flat=False, buildFaces=self.buildFaces)
  File "/home/casey/.config/blender/2.79/scripts/addons/BlenderGIS-master/operators/utils/georaster_utils.py", line 108, in exportAsMesh
    z = data[py, px]
  File "/usr/lib/python3/dist-packages/numpy/ma/core.py", line 3168, in __getitem__
    dout = self.data[indx]
IndexError: too many indices for array
```

I can open the tif fine in QGIS and see the data... (note: the above trace was [from a commit you suggested to me](https://github.com/domlysz/BlenderGIS/commit/3ea1627bf36bef2d0343652c8bcd950751165bf9) in [another issue thread](https://github.com/domlysz/BlenderGIS/issues/101); importing using the current master branch shows an equivalent of line 422 in io_import_georaster.py for that trace, and the trace is otherwise identical.)

If I do **gdal_translate example.tif output.tif**, I can import output.tif with BlenderGIS as usual and there is no error.

It appears that example.tif uses LZW compression -- could that be related? (The gdal_translate tif is not compressed, which is what inspired me to ask.)

EDIT: or maybe it's because the original tif is multi-page?

Thanks, as usual!

**Comment by domlysz** [comment]
Hi,
I can import the file without error (on Windows). I trying with step=10 otherwise I don't have enough RAM, but I make a test that reproduce the code reading the raster outside Blender (ie without building the mesh to save memory), and it also works correctly with step = 1.

Don't know what's wrong for you, maybee it's related to your linux install, is it Python 3.6 ?

**Comment by chconnor** [comment]
Thank you for checking -- 

I can't import it at any step size (including 10) unless I convert it as described (using gdal_translate).

Yes, Python 3.6:
```
>>> import sys
>>> print(sys.version)
3.6.3 (default, Oct  3 2017, 21:45:48) 
[GCC 7.2.0]
```
This is Blender 2.79b from http://ppa.launchpad.net/thomas-schiex/blender/ubuntu/

(In case it wasn't clear, this is after clicking "import raster georeferenced...".)

**Comment by s-leger** [comment]
Could be an error related to numpy setup in this repo, better download right from blender.org.

**Comment by domlysz** [comment]
yes it's seems there are some issues when running python 3.6 on Linux (#125, #109) .

I haven't yet tested this setup...

Numpy error can be solved with this couple of commands :
```
sudo apt-get install python3-pip
sudo pip3 install -U numpy
```
But I don't think the issue come from this otherwise it will raise an error that prevents addon activation.

I'll try to update my system this weekend to investigate further

**Comment by chconnor** [comment]
Great -- let me know if I can be of any help. Thanks.

**Comment by chconnor** [comment]
For the record: just upgraded to Kubuntu 18.04 which has python 3.6.6. Same issue (although I did need to install numpy to get the plugin going.)

**Comment by s-leger** [comment]
Did you try to download right from blender.org ? This could solve packaging errors often found in ppa.

**Comment by chconnor** [comment]
No, I'm using the thomas-schiex repo... Happy to try something else if it's advised, but I was going to await domlysz's advice pending whatever might get discovered if they have time to test this weekend... ?

**Comment by s-leger** [comment]
Python bundeled with 2.79b is 3.5.3, i also encounter many issues with sub packages related to numpy resulting in such array errors. 
Core devs often ask for such way to setup to prevent any ppa related errors.
Download from bf and uncompress into your documents setup pip and additionnal modules in the blender's python, and if it fails you may simply erase the downloaded folder in documents without any risk.

**Comment by domlysz** [comment]
I can confirm python 3.6 is not in cause here, it seems the issue come from PIL library. In fact, for reading image's pixel the addon can uses different library depending on what is available on the system. PIL, if available, is the second choice after GDAL and I can reproducing the issue when trying to open the file with PIL. So I've downgrading PIL rank in image engine auto-selection process, making imageio the second choice over PIL (b0ccd6d). I think it must works now, can you confirm ?

Nota : image engine selection can be setup in addon preferences
![selection_001](https://user-images.githubusercontent.com/5639024/45927549-30574e00-bf35-11e8-90f1-f0817c075c7c.jpg)

**Comment by domlysz** [comment]
yes sorry you're right, I was too hurry;

Here the correct fix b9c98ed

Thanks !

**Comment by chconnor** [comment]
Works great, thanks!
